src/components/learner-credit-management/BudgetAggregates.jsx

```diff
@@ -0,0 +1,76 @@
+import { useParams } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  Card,
+  Stack,
+  Badge,
+  Icon,
+} from '@edx/paragon';
+import { Info } from '@edx/paragon/icons';
+import { formatDate, getBudgetStatus } from './data';
+import useOffersBudgets from './data/hooks/useOffersBudgets';
+import LearnerCreditAggregateCards from './LearnerCreditAggregateCards';
+
+const BudgetAggregates = ({
+  enterpriseUUID,
+  offers,
+}) => {
+  const { budgetId } = useParams();
+  const {
+    isLoading: isLoadingBudgets,
+    allBudgets,
+  } = useOffersBudgets(enterpriseUUID, offers);
+  const budget = allBudgets?.find(item => item.id === budgetId);
+  const budgetLabel = getBudgetStatus(budget?.start, budget?.end);
+  const renderCardHeader = () => {
+    const subtitle = (
+      <Stack direction="horizontal" gap={2.5}>
+        <Badge variant={budgetLabel?.badgeVarient} className="m-0">{budgetLabel?.status}</Badge>
+        <span data-testid="offer-date">
+          {formatDate(budget?.start)} - {formatDate(budget?.end)}
+        </span>
+      </Stack>
+    );
+
+    return (
+      <Card.Header
+        title={budget?.displayName || 'Overview'}
+        subtitle={subtitle}
+      />
+    );
+  };
+  return (
+    <Card isLoading={isLoadingBudgets}>
+      {renderCardHeader()}
+      <Card.Section className="pt-0 d-flex flex-column">
+        <Stack direction="horizontal" gap={1} className="d-flex align-items-start my-3">
+          <Icon src={Info} className="mr-2 pt-1" />
+          <span className="w-75"> Data last updated on Jan 03, 2023. This data reflects the current active learner credit
```

Comment:
Curious, are we supposed to hardcode this data?

Reply:
Actually, this date value is not available in our API response now. So I am finding a way to make that value available then I will use it here.